helm_deploy/values-prod.yaml

```diff
@@ -3,6 +3,7 @@
 replicaCount: 2
 
 env:
+  APVS_AUTO_APPROVAL_ENABLED: "false"
```

Comment:
Is this right? Are we enabling it on prod?

Reply:
Doh!!